src/pages/queues-stats/index.tsx

```diff
@@ -63,49 +65,152 @@ export default function QueuesStats({
+                            <p className="font-inter text-2xl font-bold text-[#F59E0B]">{syncMapData.tasks.waitingTasks}</p>
+                        </div>
+                    </div>
+                    <div className='flex box-border rounded h-[230px] w-[135px] border-[2px] mt-5 justify-center flex-col items-center'>
+                        <div className='w-12 h-12 rounded-full bg-white border-[2px] border-gray-300 flex items-center justify-center mt-[-30%]'>
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" className="w-6 h-6 text-[#EF4444]">
+                            <path stroke-linecap="round" stroke-linejoin="round" d="M12 6v6h4.5m4.5 0a9 9 0 11-18 0 9 9 0 0118 0z" />
+                        </svg>
+                        </div>
+                        <p className='text-center font-inter text-24 font-medium leading-32 tracking-tight mb-[15%] mt-[10%]'>Longest Wait</p>
+                        <div className="w-[100px] h-[45px] bg-[#FEF2F2] rounded-full mb-[-40%] mt-[-10px] flex items-center justify-center">
+                            <p className="font-inter text-2xl font-bold text-[#EF4444]">{syncMapData.tasks.longestTaskWaitingAge}</p>
+                        </div>
+                    </div>
+                </div>
+                <h4 className="font-inter text-[18px] font-semibold leading-32 tracking-tight text-left mt-8 mb-5">Task Queues</h4>
+                <div className='table'>
+                    <div className='table-header-group'>
+                        <div className='table-row'>
+                            <div className='table-cell text-left text-[15px] font-inter'>
+                                <div className='flex'>
+                                    QUEUES 
+                                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="gray" className="w-6 h-6">
+                                        <path stroke-linecap="round" stroke-linejoin="round" d="M8.25 15L12 18.75 15.75 15m-7.5-6L12 5.25 15.75 9" />
+                                    </svg>
+                                </div>
+                            </div>
+                            <div className='table-cell text-left text-[15px] font-inter'>ACTIVE</div>
+                            <div className='table-cell text-left text-[15px] font-inter'>WAITING</div>
+                            <div className='table-cell text-left text-[15px] font-inter'>LONGEST</div>
+                            <div className='table-cell text-left text-[15px] font-inter'>AGENT STATUS</div>
+                        </div>
+                    </div>
+                    <div className='table-row-group'>
+                        <div className='table-row h-[50px]'>
+                            <div className='table-cell'>Sample Queue</div>
+                            <div className='table-cell'>
```

Comment:
Each of the columns need more spacing between elements as it looks really cramped at the moment. There are a few ways you can go about this:

1. Turn it into a flexbox with defined height and justify between or just grid-gap to add spacing between cells
2. Re-implement the table similar to [this](https://github.com/ConnieML/Connie-RTC/blob/develop/src/components/UserTable.tsx#L37) where you structure by row instead of by column
3. Set the size of each individual cell

I would recommend options 1 or 2 (preferably 2 if you have time). Let me know what you think!